LEGO1/viewmanager/viewlodlist.h

```diff
@@ -44,7 +44,7 @@ class ViewLODList : public LODList<ViewLOD> {
 // ??? for now, until we have symbol management
 typedef const char* ROIName;
 struct ROINameComparator {
-	bool operator()(const ROIName& rName1, const ROIName& rName2) const
+	int operator()(const ROIName& rName1, const ROIName& rName2) const
```

Comment:
We should leave it as `bool`, see here: https://github.com/isledecomp/isle/pull/293#discussion_r1409152942

Reply:
I see, yes it may have been only a warning.